packages/modular-scripts/src/utils/isModularType.ts

```diff
@@ -38,3 +38,16 @@ export default function isModularType(dir: string, type: PackageType): boolean {
 export function isValidModularType(dir: string): boolean {
   return ModularTypes.includes(getModularType(dir) as ModularType);
 }
+
+export function isValidModularRootPackageJson(dir: string): boolean {
+  const packageJsonPath = path.join(dir, 'package.json');
+  if (fs.existsSync(packageJsonPath)) {
+    const packageJson = fs.readJsonSync(packageJsonPath) as ModularPackageJson;
+    return (
+      packageJson.modular?.type === 'root' &&
+      !!packageJson.private &&
+      !!packageJson?.workspaces?.includes('packages/**')
```

Comment:
This is not technically a requirement, since people could also have `packages/*` or whatever workspace layout they want. We just have convention on `packages/**`. I would argue that these `private` and `workspaces` validations should be in `modular check`.

Reply:
If they ran modular init, their workspaces will have the `packages/**` convention set as their workspaces. 

This can be placed in modular check but since modular convert doesn't call check until the end, I placed it in a util and called it directly at the start

Reply:
I added the validation to modular check